win32ss/user/ntuser/simplecall.c

```diff
@@ -392,7 +392,7 @@ NtUserCallOneParam(
            break;
 
         case ONEPARAM_ROUTINE_CREATESYSTEMTHREADS:
-            Result = CreateSystemThreads(Param);
+            Result = UserSystemThreadProc(Param);
```

Comment:
Should there be a check to only allow this from csrss?

Reply:
In windows this can also be called by explorer.exe. See https://jira.reactos.org/browse/CORE-11944?focusedCommentId=109512